src/components/Icons/FullscreenIcon/FullscreenIcon.tsx

```diff
@@ -10,17 +10,10 @@ export const FullscreenIcon = LsdIcon(
       xmlns="http://www.w3.org/2000/svg"
       {...props}
     >
-      <g clip-path="url(#clip0_2418_8608)">
-        <path
-          d="M4 18C4 19.1 4.9 20 6 20H10V18H6V14H4V18ZM20 6C20 4.9 19.1 4 18 4H14V6H18V10H20V6ZM6 6H10V4H6C4.9 4 4 4.9 4 6V10H6V6ZM20 18V14H18V18H14V20H18C19.1 20 20 19.1 20 18Z"
-          fill={props.fill || 'white'}
-        />
-      </g>
-      <defs>
-        <clipPath id="clip0_2418_8608">
-          <rect width="24" height="24" fill="white" />
-        </clipPath>
-      </defs>
+      <path
+        d="M4 18C4 19.1 4.9 20 6 20H10V18H6V14H4V18ZM20 6C20 4.9 19.1 4 18 4H14V6H18V10H20V6ZM6 6H10V4H6C4.9 4 4 4.9 4 6V10H6V6ZM20 18V14H18V18H14V20H18C19.1 20 20 19.1 20 18Z"
+        fill={props.fill || 'white'}
+      />
```

Comment:
Removed the clipPath because, when there were 2 icons on the same page, 1 of them didn't show correctly. I believe this was because the clipPath id was the same for both icons, and this was creating issues. I'm not 100% sure if this was the cause though. But removing the clipPath fixed the issue.